web/themes/custom/novel/templates/paragraphs/paragraph--banner.html.twig

```diff
@@ -0,0 +1,23 @@
+{%
+  set bannerContentClasses = [
+  'banner-content arrow__hover--right-large',
+  content.field_banner_image.0 ? '' : 'banner-content--no-image'
+]
+%}
+
+<a href="{{ content.field_banner_link.0["#title"] }}" class="banner ">
```

Comment:
```suggestion
<a href="{{ content.field_banner_link.0["#title"] }}" class="banner">
```

Reply:
There is a general problem with the URL's at the moment, in that they don't support proxy'ies. [This is mentioned here](https://reload.atlassian.net/browse/DDFFORM-811). 

This applies to this current paragraph aswell. 

I suggest however you disregard that as i intend to fix it across the solution, i just wanted to mention it so you knew.